### DISCUSSION TITLE
Filter options

### DISCUSSION THREAD

**Discussion by SteveHarveyUK** [dotnet/DataGridExtensions#85] [General]
Good Afternoon Folks,

I've had a quick look around and at the documentation but haven't been able to find any examples that shows how I can alter the filter. For instance, what would the best way be to make it a 'starts with' based search rather than a contains based search?

Am I missing some good examples of use somewhere?

**Comment by tom-englert** [comment]
https://github.com/dotnet/DataGridExtensions/tree/master/src%2FDataGridExtensionsSample%2FControls

**Comment by tom-englert** [comment]
https://github.com/dotnet/DataGridExtensions/blob/master/src/DataGridExtensions/RegexContentFilter.cs
